oeps/oep-0048-brand-customization.rst

```diff
@@ -0,0 +1,88 @@
+
+=============================
+OEP-0048: Brand Customization
+=============================
+
+.. list-table::
+   :widths: 25 75
+
+   * - OEP
+     - :doc:`OEP-0048 </oeps/oep-0048-arch-brnd>`
+   * - Title
+     - Brand Customization
+   * - Last Modified
+     - 2020-10-06
+   * - Authors
+     - Adam Butterworth <abutterworth@edx.org>, Aura Milena Alba <aura.alba@edunext.co>
+   * - Arbiter
+     - Kshitij Sobti <kshitij@opencraft.com>
+   * - Status
+     - Draft
+   * - Type
+     - Architecture
+   * - Created
+     - 2020-09-09
+   * - Review Period
+     - 2020-10-12 - 2020-10-26
+   * - Replaces
+     - OEP-23 Style Customization
+
+
+Context
+-------
+
+The ability to customize the Open edX platform to reflect a custom brand and visual style is a critically important feature for the Open edX community. The comprehensive theming system built into edx-platform has served theme and branding needs for several years. It is imperfect and unofficially supported, but it has worked.
+
+Meanwhile, the edX organization has invested in a new microfrontend architecture that features many small, independently deployable microfrontend applications (MFEs). Most of the Open edX user interface will be delivered by these MFEs by the end of this technical transition.
+
+The UI in MFEs are primarily built using the Paragon SCSS and React library based on Bootstrap 4. Paragon itself supports themes via SCSS variable definitions in the same way as its parent library, Bootstrap. The edx.org theme lives inside the Paragon project along side a starter theme.
```

Comment:
Additional note: Using a JSON-only theme is also harder to support on edx-platform.

Reply:
I think we should continue our consideration of using a CSS-in-JS theming system. If we do adopt one, it should live in a brand package aside the SCSS theming system (since we're unlikely to replace it in a short period).